fwb/app/(auth)/(routes)/sign-up/[[...sign-up]]/page.tsx

```diff
@@ -26,11 +26,22 @@ export default function Page() {
   const { user } = useUser();
 
   const [error, setError] = useState<any>(null);
+
+  // Track local storage to determine if user being redirect to sign in comes from sign up page
+  const queryParams = useSearchParams();
```

Comment:
This was named searchParams in /sign-in page

Reply:
Fixed!